dashboard/src/components/UpgradeForm/UpgradeForm.v2.tsx

```diff
@@ -181,6 +184,18 @@ function UpgradeForm({
           </Column>
           <Column span={9}>
             <form onSubmit={handleDeploy}>
+              <div className="upgrade-form-version-selector">
+                <label className="centered deployment-form-label deployment-form-label-text-param">
+                  Version To Upgrade
```

Comment:
"Upgrade to Version" ?

"Version to Upgrade" implies that the displayed version is being upgrade, rather than that it's the target of the upgrade.